### PR TITLE
Create a krasmar-hometown file

### LINKED ISSUE #49: krasmar's recommendations for Prague, CZ
# Prague, Czech Republic
- [x] Create a branch
- [x] Add a file
- [ ] Commit my changes

How do you like it @vavrtom  ?

### PR DESCRIPTION
**Pull request by krasmar**
This PR closes #49 by adding a file about Prague, Czech Republic.

You can help by:

- Giving ideas of types of food you like to eat. [schnitzel? :wink: ]
- Telling me about the things you like to do

@vavrtom anything else I should add?

### CODE REVIEW

File: krasmar-hometown.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Prague, Czech Republic
```
Comment (shtldr) [inline krasmar-hometown.md]: Mas tam chybu, voe :)

Comment (krasmar) [inline krasmar-hometown.md]: nevidim

**Comment by shtldr** [comment]
Nic moc...

**Comment by vavrtom** [comment]
Moc tech mist tam neni :D

**Comment by krasmar** [comment]
Jak rika Vasek Klausu: "A vy jste kdo?"

**Comment by shtldr** [comment]
To si jako z prace udelame Twitter nebo co? To mi tady jeste chybelo...

File: krasmar-hometown.md (hunk 1 above)
Comment (shtldr) [inline krasmar-hometown.md]: Delal jsem si srandu

**Comment by tuaminx** [comment]
just for test :smile:

File: krasmar-hometown.md [hunk 2]
```diff
@@ -7,5 +7,7 @@ http://nasemaso.ambi.cz/en/
 https://en.wikipedia.org/wiki/Prague_Castle
 ## Places to avoid
 https://www.tripadvisor.com/ShowUserReviews-g274707-d1948084-r250059814-Restaurace_Klementinum-Prague_Bohemia.html
+## Things to avoid
+Do not eat yellow snow.
```
Comment (shtldr) [inline krasmar-hometown.md]: Ty prase! :-1:

**Pull request merged**